tools/cordova/index.js

```diff
@@ -13,7 +13,7 @@ export const CORDOVA_ARCH = "web.cordova";
 
 export const CORDOVA_PLATFORMS = ['ios', 'android'];
 
-const CORDOVA_ANDROID_VERSION = "10.1.2";
+const CORDOVA_ANDROID_VERSION = "12.0.0";
 
 export const CORDOVA_DEV_BUNDLE_VERSIONS = {
```

Comment:
Please update these as per #12518

Reply:
```
'cordova-lib': '12.0.1',
  'cordova-common': '5.0.0',
  'cordova-create': '5.0.0',
```

Reply:
This PR won't update these ones, I added the explanation on the PR description, but basically:

> The cordova-lib, cordova-create and cordova-common were not updated because updating it breakes the build process. For some reason the node_modules on the resulting cordova project is not correctly populated, and the main java file that extends CordovaActivity is not created.
> However, those modules are related to the meteor-tool only, so not updating it shouldn't affect anything.